test/rpc/test_rpc_misc.py

```diff
@@ -121,3 +150,25 @@ def test_call_with_invalid_params(params):
     # could be any legal method, just passing something to get params to fail
     ex = rpc_call(method="starknet_getClass", params=params)
     assert ex["error"] == {"code": -32602, "message": "Invalid params"}
+
+
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "run_devnet_in_background, input_data, expected_data",
+    GET_EVENTS_TEST_DATA,
+    indirect=True,
+)
+def test_get_events(input_data, expected_data):
+    """
+    Test RPC get_events.
+    """
+    deploy_info = deploy(EVENTS_CONTRACT_PATH)
+    for i in range(0, 2):
+        invoke(
+            calls=[(deploy_info["address"], "increase_balance", [i])],
+            account_address=PREDEPLOYED_ACCOUNT_ADDRESS,
+            private_key=PREDEPLOYED_ACCOUNT_PRIVATE_KEY,
+        )
+    resp = rpc_call("starknet_getEvents", params=input_data)
```

Comment:
What if returned data contains more elements than expected? Since iterating over expected data, that will not be registered, right? Perhaps adding a length assertion would make sense.

Reply:
fixed